uber/configspec.ini

```diff
@@ -49,6 +49,11 @@ use_checkin_barcode = boolean(default=False)
 # registering, False to disallow promo codes.
 badge_promo_codes_enabled = boolean(default=False)
 
+# This controls whether the "Request Hotel Booking Info" step is including in
+# the preregistration workflow. True to include the "Request Hotel Booking
+# Info" step during preregistration, False to remove it.
+prereg_request_hotel_info_enabled = boolean(default=False)
```

Comment:
Whoa, whoa, whoa! Just had an idea...

What if this was a duration instead of a date. We would define an interval of time after preregistration opens during which the request checkbox will be available (like 24 hours, or 2 days, or something, but probably expressed as a number of seconds).

That way we don't have to change two dates (`prereg_open` and `prereg_request_hotel_info_enabled`) to configure this correctly.